src/video_core/renderer_opengl/renderer_opengl.cpp

```diff
@@ -150,6 +150,8 @@ void RendererOpenGL::SwapBuffers() {
     {
         auto aggregator = Common::Profiling::GetTimingResultsAggregator();
         aggregator->AddFrame(profiler.GetPreviousFrameResults());
+        if (VideoCore::g_toggle_framelimit_enabled)
+        FrameLimiter(aggregator->GetAggregatedResults().frame_time.avg, 60);
```

Comment:
okay will try that.i still prefer using float for the ratio 1000/60 , which is 1.66666... ..though to prevent warnings i would have to find a common multiplier and pass it as uint32_t.